app/src/main/res/values/strings.xml

```diff
@@ -354,6 +354,9 @@
     <string name="add_beneficiary_option">Add Third Party Beneficiaries to your account.\nEither
         enter manually or scan account QR code
     </string>
+    <string name="income_generating_loan">Income Generating Loan</string>
+    <string name="not_applicable">Not Applicable</string>
+    <string name="principal_amount_number">143.00</string>
```

Comment:
Isn't it set through java later? Do we need this?

Reply:
As of now it isn't edited by java.

Reply:
Yes, removed it. Thanks

Reply:
Yes, modified it. Thanks!